entrypoints/puppet-lint

```diff
@@ -37,7 +37,16 @@ eval $(pdk env)
 if [[ $manifests -eq 0 ]] || [[ ! -s lint_output.json ]]; then
   echo 'null' > lint_output.json
 fi
-ruby -e "require 'json'; puts ({ exit_code: $exit_code, manifests: $manifests, lines: $lines, output: JSON.parse(File.read('lint_output.json')) }).to_json" > anubis_output.json
+
+linter_version=$(puppet-lint --version | grep -oP '\d\.\d\.\d')
```

Comment:
Do we need `\d+` here in case one of the components goes to 2 digits?

Reply:
Oh, good call! I will add that.